### PR TITLE
Remove unused dependencies and files, optimise imports/exports

### PR DESCRIPTION
**Pull request by timoheddes**
- Found and removed unused dependencies, files, imports and exports using [Knip](https://www.npmjs.com/package/knip).
- Updated TS path aliases and simplified the style related imports/exports.

### CODE REVIEW

File: packages/libs/react-ui/package.json [hunk 1]
```diff
@@ -99,18 +89,16 @@
     "eslint-plugin-import": "~2.27.5",
     "eslint-plugin-storybook": "~0.6.13",
     "jest": "~29.6.1",
-    "jest-environment-jsdom": "~29.6.1",
     "jest-standard-reporter": "~2.0.0",
     "mini-css-extract-plugin": "2.7.6",
     "prettier": "~3.0.0",
-    "prettier-plugin-packagejson": "^2.4.5",
+    "knip": "~2.24.1",
```
Comment (eileenmguo) [inline packages/libs/react-ui/package.json:95]: should this be here? I think it is only needed at the root level?

Comment (timoheddes) [reply packages/libs/react-ui/package.json:95]: Oh yeah I guess so

File: package.json [hunk 2]
```diff
@@ -21,10 +21,10 @@
   "devDependencies": {
     "@changesets/cli": "^2.26.2",
     "@kadena-dev/markdown": "workspace:*",
-    "knip": "^2.21.2",
+    "knip": "^2.25.2",
     "only-allow": "^1.1.1",
     "syncpack": "^10.9.3",
-    "tsx": "^3.12.8",
+    "tsx": "^3.12.10",
```
[the remark this answers is not in the document]
Comment (timoheddes) [reply package.json]: Reverted that change

[the remark this answers is not in the document]
Comment (webpro) [reply package.json]: Thanks!

Surprised this PR is a net positive in terms of LOC though:

Screenshot 2023-09-21 at 11 18 39 AM

**Review by webpro — APPROVED** [review]

**Review by sstraatemans — APPROVED** [review]

**Pull request closed without merging**